<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="artist_input">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Artist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="album_input">
       <property name="placeholderText">
        <string>Album</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="rday_input">
         <property name="placeholderText">
          <string>day</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rmonth_input">
         <property name="placeholderText">
          <string>month</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ryear_input">
         <property name="placeholderText">
          <string>year</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="genre_input">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="title_input">
       <property name="placeholderText">
        <string>Titel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="apply_push">
       <property name="text">
        <string>Apply Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="repeat_push">
       <property name="text">
        <string>Download next</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
